src/platform/packages/shared/kbn-scout/src/playwright/fixtures/worker/scout_space/parallel.ts

```diff
@@ -130,11 +118,12 @@ export const scoutSpaceParallelFixture = coreWorkerFixtures.extend<
         setDefaultTime,
       };
 
-      log.serviceLoaded(`scoutSpace:${spaceId}`);
+      log.serviceLoaded(`new Kibana space ${spaceId} created`);
```

Comment:
Now that we updated the `serviceLoaded` function to print `loaded` at the end of the string (thanks for making the change), this may not make much sense anymore (the log would end with `created loaded`). Perhaps we want to use `debug` here?

Reply:
you got me again :) as it is important to know space name to better understand error logs / screenshot, we can use 

```
serviceMessage('scoutSpace', `new Kibana space '${spaceId}' created`)
```

to see 

```
 debg [scout-worker-1] [scoutSpace] new Kibana space 'test-space-1' created

```